Sources/Controllers/2.0/Settings/ImportExport/OAExportItemsSelectionViewController.mm

```diff
@@ -33,9 +33,13 @@
 #import "OAGlobalSettingsItem.h"
 #import "OADestination.h"
 #import "OAPOIHelper.h"
+#import "OATableViewCustomHeaderView.h"
 
 #include <OsmAndCore/ArchiveReader.h>
 
+#define titleWithDescrCellHeight 60.0
+#define kHeaderId @"TableViewSectionHeader"
```

Comment:
This is an old way of registering classes for tableview. Please use the new method that utilizes the static method for each cell and header ```getCellIdentifier```